catalog/core/catalog-core-federationstrategy/pom.xml

```diff
@@ -48,7 +48,10 @@
         <dependency>
             <groupId>ddf.catalog.core</groupId>
             <artifactId>catalog-core-api-impl</artifactId>
-            <type>test-jar</type>
+            <type>jar</type>
+            <!-- Bug in dependency-plugin 3.0.2 requires using classifier for test-jars instead of type.
```

Comment:
thanks for adding the comments in here

Reply:
its the type of thing that I myself would have changed back if I didn't see a comment like that 😄

Reply:
yes, I would have forgotten by then